content/en/synthetics/mobile_app_testing/mobile_app_tests/restricted_networks.md

```diff
@@ -41,6 +41,19 @@ The following is the list of IP ranges associated with the real devices used for
 `185.94.24.0/22`</br>
 `34.96.70.78`</br>
 
+The following is the list of IP ranges associated with Synthetics Mobile HTTP steps. Please disregard if your company does not use this feature.
```

Comment:
nit: maybe change "company..." to "tests do not use this feature"? A single company could have more than one Datadog org/more than one mobile app that they're testing etc.